### PR TITLE
Feature/implement new home screen

### PR DESCRIPTION
**Pull request by victorienfotsing**

### CODE REVIEW

**Comment by victorienfotsing** [comment]
#194 #196

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "coverage": "vitest run --coverage",
     "e2e": "cypress open",
     "lint": "stylelint \"**/*.{vue,scss}\" && prettier . --check && eslint --ext ts,js,vue,json . && ELECTRON=true nuxi typecheck",
+    "lint:fix": "stylelint --fix \"**/*.{vue,scss}\" && prettier . --check --write && eslint --fix --ext ts,js,vue,json .",
```
Comment (SimonSimCity) [inline package.json:19]: Could you please check first what combining the options `--check` and `--write` does to the prettier command?

**Review by SimonSimCity — COMMENTED** [review]
Looks good 😎 Please confirm about the lint-fix command you've added.

File: package.json (hunk 1 above)
Comment (victorienfotsing) [reply package.json]: I have removed the option `--check` since it is useless for lint fix

**Pull request merged**